### PR TITLE
Covers MUWM-4923 task 1, 3, and 4 

### PR DESCRIPTION
**Pull request by SarthakSingh31**

### CODE REVIEW

File: myuw/static/vue/components/_common/visual_schedule/schedule-tab.vue [hunk 1]
```diff
@@ -194,7 +193,7 @@ export default {
   },
   computed: {
     ...mapState({
-      today: (state) => dayjs(state.termData.todayDate),
+      today(state) { return this.dayjs(state.termData.todayDate); },
```
Comment (fanglinfang) [inline myuw/static/vue/components/_common/visual_schedule/schedule-tab.vue:196]: we have nowDatetime in mixins/utils, do we want to use that?

File: myuw/static/vue/components/_common/visual_schedule/schedule.vue [hunk 2]
```diff
@@ -70,7 +69,7 @@ export default {
   computed: {
     ...mapState({
       allSchedules: (state) => state.visual_schedule.value,
-      today: (state) => dayjs(state.termData.todayDate),
+      today(state) { return this.dayjs(state.termData.todayDate); },
```
Comment (fanglinfang) [inline myuw/static/vue/components/_common/visual_schedule/schedule.vue:72]: we already have nowDatetime in mixins/utils

File: myuw/static/vue/components/accounts/tuition-fees.vue [hunk 3]
```diff
@@ -280,7 +278,7 @@ export default {
     },
     tuitionDate() {
       // from notice
-      const now = this.nowDatetime();
+      const now = this.nowDatetime(false);
```
Comment (fanglinfang) [inline myuw/static/vue/components/accounts/tuition-fees.vue:281]: we may not want to change it to false.

File: myuw/static/vue/components/home/summaries.vue [hunk 4]
```diff
@@ -204,7 +201,7 @@ export default {
         return parseInt(days / 7) + 1;
       }
     },
-    toFromNowDate: (s) => dayjs(s).fromNow(),
+    toFromNowDate(s) { return this.dayjs(s).fromNow(); },
```
Comment (fanglinfang) [inline myuw/static/vue/components/home/summaries.vue:204]: Is fromNow based on the cardDisplayDates.comparison_date?

File: myuw/static/vue/components/accounts/tuition-fees.vue (hunk 3 above)
Comment (SarthakSingh31) [reply myuw/static/vue/components/accounts/tuition-fees.vue:281]: false makes it so it never uses the override date

File: myuw/static/vue/plugins/logger.js [hunk 5]
```diff
@@ -153,11 +153,17 @@ class Logger {
     });
   }
 
-  classEmailList(component, cardTid) {
-    this.sink.event('class_email_list', {
+  classEmailList(component) {
```
Comment (fanglinfang) [inline myuw/static/vue/plugins/logger.js:156]: Could we add a param "action", whose value is the innerHtml?

**Review by fanglinfang — APPROVED** [review]

**Pull request merged**